### ISSUE TITLE
Import Format - Batch Normalization

### ISSUE THREAD

**Issue by pedr0sorio** [JihongJu/keras-resnet3d#12]
It seems that there is an issue when importing  'BatchNormalization' from 'keras.layers.normalization' in the script [resnet3d.py]

The import should be done in a different way: 

_from tensorflow.keras.layers import BatchNormalization_ 

as opposed to 

 _from keras.layers.normalization import BatchNormalization_

Maybe the issue is due to some underlying problem.

I am installing this package with the command below in google colab

`! pip install git+https://github.com/JihongJu/keras-resnet3d.git
`
and creating a modle like so:
```
from resnet3d import Resnet3DBuilder
model = Resnet3DBuilder.build_resnet_50((96, 96, 96, 1), 20)
```

**Comment by JihongJu** [comment]
Hi @pedr0sorio, the package was made in the Keras 1.0 era. Feel free to send a pull request to make it Tensorflow 2.0 compatible.
